bitnami-services/upgrading-tkr.hbs.md

```diff
@@ -0,0 +1,143 @@
+# Upgrading to TKR 1.26 or newer
+
+The newest version of TKR enforce a [`restricted` Pod Security
+Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) for all pods
+running on the cluster. This will have an impact on your running Bitnami services. In this topic we explore
+possible solutions to this problem.
```

Comment:
```suggestion
two possible solutions to this problem. Where possible, it is recommended to follow the first solution, which involves upgrading to TAP 1.7 and then updating CompositionRevision references for any existing Bitnami Services instances. See the "If upgrading to TAP 1.7" section below. If you do not wish or are unable to upgrade to TAP 1.7 for any reason, then an alternative solution is provided. See the "If staying on TAP 1.6 or older" section below.
```

Reply:
Do we keep this document the same and backport it to every version, or do we have different documents for 1.7 and for 1.6 or older? I assume if someone is reading the 1.7 docs they're running on 1.7?